protocols/v2/noise-sv2/src/initiator.rs

```diff
@@ -252,6 +351,12 @@ impl Initiator {
         }
     }
 
+    /// Securely erases sensitive data in the initiator's memory.
+    ///
+    /// Clears all sensitive cryptographic material within the [`Initiator`] to prevent any
+    /// accidental leakage or misuse. It overwrites the stored keys, chaining key, handshake hash,
+    /// and session ciphers with zeros or other non-sensitive values. This method is typically
```

Comment:
what is `or other non-sensitive-values`?

Reply:
I agree this is a bit unclear. We should try to clarify the wording here.

Reply:
Currently for reseting the state, we are zeroing them out. But we can also use some non-sensitive garbage value. I will remove this non-sensitive values as well.